dandiapi/api/models/version.py

```diff
@@ -83,15 +83,32 @@ def make_version(cls, dandiset: Dandiset = None) -> str:
     def copy(cls, version):
         return Version(dandiset=version.dandiset, metadata=version.metadata)
 
+    @classmethod
+    def citation(cls, metadata):
+        year = datetime.datetime.now().year
+        name = metadata['name']
+        url = metadata['url']
+        if 'contributor' not in metadata:
```

Comment:
For when `metadata['contributors']` exists, but is an empty list
```suggestion
        if 'contributor' not in metadata or not metadata['contributor']:
```